### PR TITLE
docs: add tresjs (vue) to docs

### LINKED ISSUE #43: TresJs implementation
Hi! would you accept a PR,  to contain the implementation for using CSM with TresJs?

[TresJs](https://github.com/Tresjs) is the R3F of Vue.

I have already playing with your library but using the vainilla version [example](https://lab.jaimetorrealba.com/CSMDemo)
The import will do like:
import CustomShaderMaterial from 'three-custom-shader-material/tresjs' (or similar)

And you can also be use easily with the Vue 3D comunity :)

### PR DESCRIPTION
**Pull request by JaimeTorrealba**
closes #43 

Hi, I wanted to create the implementation here in this pkg but in the end another user have done in the Cientos pkg (our drei version). So if you agree I would like to let the people in the docs

Any feedback or comment please let me know

### CODE REVIEW

File: package/README.md [hunk 1]
```diff
@@ -100,6 +100,29 @@ function Cube() {
 }
 ```
 
+</details>
+<details>
+  <summary>Show Vue (Tresjs) example</summary>
```
Comment (FarazzShaikh) [inline package/README.md:105]: Can you please add an installation step for cientos inside the Vue dropdown?

File: package/README.md [hunk 2]
```diff
@@ -222,6 +245,43 @@ function Cube() {
...
+    <CustomShaderMaterial
+      :base-material="new CustomShaderMaterialImpl({
+        baseMaterial: materialBase.value,
+      })"
+    />
+  </TresMesh>
+</template>
+```
+
+  </details
```
Comment (FarazzShaikh) [inline package/README.md:283]: I believe this is missing a “>”, breaking the markdown for the following line

**Review by FarazzShaikh — CHANGES_REQUESTED** [review]
Thanks for the PR! Looking great, just a couple minor changes 😄

File: package/README.md (hunk 1 above)
Comment (JaimeTorrealba) [reply package/README.md:105]: More than an installation step, Would be better to let them know that they need Cientos for this one, don't you think?

**Review by FarazzShaikh — APPROVED** [review]
Great! Thanks! Im gonna piggyback a few of my own readme updates to this PR :)

**Pull request merged**